pydfc/data_loader.py

```diff
@@ -242,16 +303,33 @@ def nifti2timeseries(
     it uses nilearn maskers to extract ROI signals from nifti files
     and returns a TIME_SERIES object
 
-    For now it only works with schaefer atlas,
-    but you can set the number of rois to extract
+    It will extract the mean time series from each ROI defined either in mask_file
+    which should be provided as a 3D volumetric parcellation in the same space
+    as the subject/session nifti file (if mask_file is assigned) or will extract
+    the mean time series from ROIs extracted from the Schaefer atlas, if
+    you set the number of rois to extract
     {100, 200, 300, 400, 500, 600, 700, 800, 900, 1000}
     """
-    time_series, labels, locs = nifti2array(
-        nifti_file=nifti_file,
-        confound_strategy=confound_strategy,
-        standardize=standardize,
-        n_rois=n_rois,
-    )
+    if mask_file is None and n_rois is None:
+        print("Either mask_file or n_rois must be defined.")
```

Comment:
it's better to raise an error here:
```suggestion
    if mask_file is None and n_rois is None:
        raise ValueError("Either mask_file or n_rois must be defined.")
```